test/main.test.js

```diff
@@ -719,3 +722,19 @@ describe('bundling multiple css feeds', () => {
 
     afterEach(() => server.close());
 });
+
+describe('unknown asset types', () => {
+    let server;
+    let post;
+
+    beforeEach(async () => {
+        const router = new Router();
+        ({ server } = await createTestServerFor(router.router()));
+
+        ({ post } = supertest(server)); // eslint-disable-line prefer-const
+    });
+
+    test('unknown filetype gives 404', () => post('/bundle/png').expect(404));
```

Comment:
404 keeps the old behavior. Could be a 400, I suppose? IDK

Reply:
yeah agree could be either but unless you have strong feelings towards 400 I'd say lets just keep it the same and stick with 404?